### PR TITLE
Return Megaphone photo URL in API response

### PR DESCRIPTION
**Pull request by rikonen**

### CODE REVIEW

File: helpers/models_helper.rb [hunk 1]
```diff
@@ -1345,6 +1345,10 @@ def megaphone_model
         :created_at => {
           :type => Const::DATE_TIME,
           :required => true
+        },
+        :image_url => {
+          :type => Const::STRING,
+          :required => true
```
Comment (Locke23rus) [inline helpers/models_helper.rb]: @rikonen attribute `image_url` is optional, so we should drop attribute `required` here ;)

Comment (rikonen) [inline helpers/models_helper.rb]: The field is always returned, it just might be `null`. I thought required here was referring to whether the field can be expected to be present or not. But it then distinguishes between null and non-null fields?

Comment (Locke23rus) [inline helpers/models_helper.rb]: Yes in this context it used as required/optional. Swagger will generate for us `image_url (string, optional)`

Comment (rikonen) [inline helpers/models_helper.rb]: OK, fixed.

**Comment by Locke23rus** [comment]
:+1:

**Pull request merged**